Sources/NIOHTTP2/HTTP2ChannelHandler.swift

```diff
@@ -486,7 +493,16 @@ extension NIOHTTP2Handler {
     /// Emit any pending user events.
     private func processPendingUserEvents(context: ChannelHandlerContext) {
         for event in self.inboundEventBuffer {
-            context.fireUserInboundEventTriggered(event)
+            switch event {
+            case .streamCreated(let event):
+                self.demultiplexer?.streamCreated(event.streamID, localInitialWindowSize: event.localInitialWindowSize, remoteInitialWindowSize: event.remoteInitialWindowSize)
+            case .initialStreamWindowChanged(let event):
+                self.demultiplexer?.initialStreamWindowChanged(by: event.delta)
+            case .streamWindowUpdated(let event):
+                self.demultiplexer?.streamWindowUpdated(event.streamID, inboundWindowSize: event.inboundWindowSize, outboundWindowSize: event.outboundWindowSize)
+            case .streamClosed(let event):
+                self.demultiplexer?.streamClosed(event.streamID, reason: event.reason)
+            }
```

Comment:
Can we hide this switch in an extension on the demultiplexer? It'll again let us clean up this call site somewhat.

Reply:
I have moved this to:
```
extension NIOHTTP2Handler.ConnectionDemultiplexer {
    func process(event: InboundEventBuffer.BufferedHTTP2UserEvent) {
 ```